SCClassLibrary/Common/Audio/Splay.sc

```diff
@@ -39,18 +54,21 @@ SplayAz : UGen {
 	*kr { arg numChans = 4, inArray, spread = 1, level = 1, width = 2, center = 0.0, orientation = 0.5, levelComp = true;
 
 		var n = max(1, inArray.size);
-		var pos = if(n == 1) { center } { [ center - spread, center + spread ].resamp1(n) };
-		if (levelComp) { level = level * n.reciprocal.sqrt };
+		var normSpread = (n - 1 / n) * spread;
+		var pos = if(n == 1) { center } { [ center - normSpread, center + normSpread ].resamp1(n) };
+		level = level * LevelComp(levelComp, \control);
```

Comment:
`n` is not passed to `LevelComp`!

Found when testing:
```
{ Splay.ar(Saw.ar([20, 30, 50]), 1, levelComp: true) }.scope;
```
Which throws an error, calling `reciprocal` on `nil` (n)

Reply:
oh wow, no idea how that sneaked back in - thanks!